src/components/RadioButtonGroup/styled/buttonItem.js

```diff
@@ -1,8 +1,30 @@
 /* stylelint-disable max-line-length */
 import styled from 'styled-components';
+import { COLOR_GRAY_1, COLOR_GRAY_2 } from '../../../styles/colors';
 import { BORDER_RADIUS_2 } from '../../../styles/borderRadius';
+import getTheme from '../../../styles/helpers/getTheme';
 
-const StyledButtonItem = styled.span`
+const StyledButtonItem = styled.span.attrs(props => {
+    const theme = getTheme(props);
+    const { getContrastText, brand, success, error } = theme.palette;
+    const { main: brandMainColor, dark: brandDarkColor } = brand;
+    const { main: successMainColor, dark: successDarkColor } = success;
+    const { main: errorMainColor, dark: errorDarkColor } = error;
+
+    return {
+        brandMainColor,
+        brandDarkColor,
+        successMainColor,
+        successDarkColor,
+        errorMainColor,
+        errorDarkColor,
+        getContrastText,
+        // TODO: move up to defaultTheme or normalizeTheme
+        brandShadow: `0 0 2px ${brandMainColor}`,
+        successShadow: `0 0 2px ${successMainColor}`,
+        errorShadow: `0 0 2px ${errorMainColor}`,
```

Comment:
here only use what is needed, for example successMainColor and errorMainColor are not being used